mysql-test/suite/encryption/t/innodb-bad-key-change3.test

```diff
@@ -1,10 +1,7 @@
---skip #has import
 -- source include/not_valgrind.inc
 
-call mtr.add_suppression("cannot be decrypted. Are you using correct keyring that contain the");
-call mtr.add_suppression("InnoDB: Tablespace for table .* is set as discarded.");
-call mtr.add_suppression("InnoDB: Cannot calculate statistics for table .* because the .ibd file is missing. Please refer to .* for how to resolve the issue.");
-
+call mtr.add_suppression("\\[InnoDB\\] Tablespace id 4 in file t1\.ibd is encrypted but keyring or used key_id 4 is not available\. Can't continue reading table\. Please provide the correct keyring\.");
```

Comment:
Is the tablespace ID stable if the same worker instance has run more or fewer tests before this one?

Reply:
I think it should be stable. All tests have to pass MTR validation that they leave the server in the same state that they found it. Thus all tables created by a test should have been removed by it. Then the tablespace id should be always the same - in theory ...

Reply:
I think it should be stable. All tests have to pass MTR validation that they leave the server in the same state that they found it. Thus all tables created by a test should have been removed by it. Then the tablespace id should be always the same - in theory ...

Reply:
And are the freed tablespace IDs reused?

Reply:
Ah, right, they are not reused, fixed.